### PR TITLE
Add grab-points to climbing

### PR DESCRIPTION
**Pull request by Malcolmnixon**
This PR adds grab-point support to climbable objects.

https://github.com/GodotVR/godot-xr-tools/assets/1863707/25df6559-37bf-4dd5-b4fc-8327ba937f5b

### CODE REVIEW

**Comment by DigitalN8m4r3** [comment]
tested, lgtm

File: VERSIONS.md [hunk 1]
```diff
@@ -8,7 +8,11 @@
 - **breakage** Added support for two-handed grabbing.
 - Added support for snapping hands to grab-points.
 - Added support for world-grab movement.
+<<<<<<< Updated upstream
```
Comment (BastiaanOlij) [inline VERSIONS.md:11]: Merge issue ended up here :)

**Review by BastiaanOlij — APPROVED** [review]
lgtm

File: VERSIONS.md (hunk 1 above)
Comment (Malcolmnixon) [reply VERSIONS.md]: Whups - fixed.

**Pull request merged**